docs/examples.rst

```diff
@@ -53,3 +62,10 @@ Will map to the data (and vice versa):
 
 Note the SINGLE keyword is turned into a field, but the DOWN row is turned into a list.
 People can add as many or as few items to the DOWN table as they want.
```

Comment:
```suggestion
Users can add as many items to the DOWN table as they want.
```

Reply:
I know Users is kinda industry standard term but it's also not a great term. I'd prefer to leave it. If we are going to clarify this at all, we should acknowledge that People/Users could refer to one of several groups of people and we should say which. Something like "The people filling in the spreadsheet can ..."

Reply:
"The people filling in the spreadsheet can..." sounds good, just "People..." is too informal and isn't clear about *which* people.